taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      </p>
+      <h4 id="scp_sftp_rsync">
+        <code>scp</code>, <code>sftp</code>, &amp; <code>rsync</code>
+      </h4>
+
+      <p>
+        These three command line tools are secure and can be used to accomplish data transfer. You can run these commands directly from the terminal if your local system runs Linux or macOS.
+      </p>
+      <blockquote>
+        <p>
+          Note: It is possible to use these command line tools if your local machine runs Windows, but you will need to use a ssh client (ex.
+          <a target="_blank" href="https://www.putty.org/">PuTTY</a>
+          ).
+        </p>
+        <p>
+          However, for the scope of this project and ease in data transfer, it is recommended that Windows users follow our
+          <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a>
+          guide below.
+        </p>
+      </blockquote>
+      <p>
+        For users that are new to the command line, using either <code>scp</code> or <code>sftp</code> to transfer data is advised.
+      </p>
+      <h4 id="data_transfer_prereqs">
+        Prerequisites for Data Transfer with Command Line Tools
+      </h4>
+
+      <p>Before we begin, you will need to know:</p>
+
+      <ul>
+        <li>the path to your data file(s) on your local system</li>
+        <li>the path to your transfer directory on the remote storage server</li>
+      </ul>
+
+      <h4 id="local_data_path">
+        Determining the Path to Your Data File(s) on Your Local System
+      </h4>
+
+      <p>
+        In order to transfer your project data, you will need to know where your file(s) is located on your local system. To do so, navigate through the file system on your local computer.
+      </p>
+      <p>
+        For example, you can do so on Mac by going to Finder. Common data file locations such as Desktop and Documents should be accessible through the Favorites pane on the left-hand side of the window. You can also do Finder &gt; Go to find other common locations on your computer.
+      </p>
+      <p>
+        Once you have navigated to where your file(s) is located, take note of its path by right-clicking on the file and selecting Get Info.
+      </p>
+      <figure id="figure1">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6c/ea/6ceacba9-7824-4925-b924-dd317040b629/picture1.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 1. Use Get Info to determine “Where” the path of your data file(s) is
+        </figcaption>
+      </figure>
+
+      <p>
+        For example, a file located in an <code>portal-data</code> folder in
+        <code>Documents</code> would have the following path:
+      </p>
+      <p>
+        <code>/Users/myname/Documents/portal-data/my_file.txt</code>
+      </p>
+      <h4 id="storage_data_path">
+        Determining the Path to Your Transfer Directory
+      </h4>
+
+      <p>
```

Comment:
A transfer directory on the remote storage server associated with the portal you are accessing it through will be established when your account is given access to the portal and completes the on-boarding procedure. The transfer directory path will be unique for every institution and project.